en/components/grids_templates/excel_style_filtering.md

```diff
@@ -337,6 +337,347 @@ If you want to keep the sorting, moving, pinning and hiding features of the colu
 
 <div class="divider--half"></div>
 
+###Styling
+
+To get started with styling the filtering row, we need to import the `index` file, where all the theme functions and component mixins live:
```

Comment:
We are styling the **Excel Style Filtering**, not the **filtering row**

Reply:
Addressed.